crypto/ake/enclave/src/lib.rs

```diff
@@ -168,7 +169,7 @@ impl<EI: EnclaveIdentity> AkeEnclaveState<EI> {
         let report_body = Report::new(None, None)?.body();
         let initiator = Start::new(
             peer_id.to_string(),
-            report_body.mr_enclave(),
+            vec![report_body.mr_enclave()],
```

Comment:
This can can very likely be `&[report_body.mr_enclave()]`, and should be declared on the stack as an array if not. There's no need for heap here.